entity-utils/src/main/java/eu/europeana/entity/utils/EntityUtils.java

```diff
@@ -48,4 +48,21 @@ public static List<String> updateBaseUrlInIds(List<String> entityUris, String da
         }
         return updatedUris;
     }
+    
+    public static String validateUriAndRemoveQuotes(String uri) {
```

Comment:
the response of the method is not clear, please add javadoc to clearly indicate what the method does